EIPS/eip-2935.md

```diff
@@ -71,11 +71,12 @@ def resolve_blockhash(block: Block, state: State, arg: uint64):
 Exact evm assembly that can be used for the contract to resolve `BLOCKHASH`
 
 ```
-// check if inputsize>8 revert
-push1 0x08
-calldatasize
+// check if input > 8 byte value revert
+push8 0xffffffff
+push0
+calldataload
 gt
-push1 0x31
+push1 0x39
```

Comment:
ok, you removed 3 instructions and added 11, that's 8 indeed.

Reply:
yea, but i just checked the new jumpdest destinations via evm codes :) but yes math checks out